app.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { sendTopics } = require("./controllers/topics.controllers");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
We can remove this for now as it is not required just yet :) not until we get to posting